docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from pylab import rcParams
</pre><p>This line can be removed </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>OK, I will. I fogot to do this.</p>